src/reducers.js

```diff
@@ -0,0 +1,32 @@
+const initialState = {
+    message: 'dziala!',
+    time: new Date(),
+    users: [],
+};
+
+const reducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'GET_CURRENT_TIME':
+            return { ...state, time: new Date() };
+        // updateTime() // mogę to obsłuyć poza reduktorem?
```

Comment:
Nie za bardzo rozumiem, co chcesz obsłużyć, ale raczej nie jest to dobry pomysł.
To właśnie reducer ma być taką czarną skrzynką, która zarządza stanem.
W następnym materiale będzie `thunk`, który może rozwiązać Twój problem :)